### ISSUE TITLE
Typo in `NewLineBetween`

### ISSUE THREAD

**Issue by zeraye** [Jacalz/linedisp#2]
https://github.com/Jacalz/linedisp/blob/b1195e01d767c198fa315f6135299f2ff30c42b0/lines.go#L135-L142

I believie `x2` and `y1` are swapped and it should be as follows:

```
Position1:   fyne.NewPos(float32(x1)+300, float32(y1)+300),
Position2:   fyne.NewPos(float32(x2)+300, float32(y2)+300),
```

**Comment by Jacalz** [comment]
The code is correct. It is drawing lines between the x-values and y-values specifically and not points in a usual sense. If you implement the change locally, you will see that it breaks the output quite a bit:
![image](https://user-images.githubusercontent.com/25466657/234633929-6cf52683-4861-4ca8-b8ab-34201bba6fa3.png)

**Comment by zeraye** [comment]
I write this to clear things up. Issue description can be misinterpreted because of lack of information.

Original code works properly, but naming convention semantically doesn't make sense. Function `NewLineBetween` takes `x1, y1, x2, y2` which you can understand as:
`x1, y1` - coordinates of first point
`x2, y2` - coordinates of second point
thus
```
Position1:   fyne.NewPos(float32(x1)+300, float32(x2)+300),
Position2:   fyne.NewPos(float32(y1)+300, float32(y2)+300),
```
doesn't make sense, because here you use `x1`, `x2` as coordinates for first point and `y1`, `y2` for second.

But when you look at function `LinesFromMatrix`, let's for example take line
`NewLineBetween(M[0][5], M[0][11], M[1][5], M[1][11]),`.

Let's use shortcut and 0-coordinate is x, 1-coordinate is y

Arguments are:
x from 5th point,
x from 11th point,
y from 5th point,
y from 11th point.

but function `NewLineBetween` takes `x1, y1, x2, y2` NOT `x1, x2, y1, y2`.

To sum up in my opinion this naming convetion is misleading, and you can change it by:
1) `NewLineBetween` takes `x1, x2, y1, y2` and apply fix from the issue description
2) in function `LinesFromMatrix` where you use `NewLineBetween` change order of arguments
e.g. `NewLineBetween(M[0][5], M[1][5], M[0][11], M[1][11]),` and apply fix from the issue description.

**Comment by Jacalz** [comment]
Ah, I see. Makes sense. Thanks for the clarification. Pull requests are always appreciated ;)
